### ISSUE TITLE
Explore dashboard opportunities in Domo using Github data

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#25770]
## Issue Description
We are preparing to launch our new VA.gov Experience Standards and process changes to VFS teams. We'd like to ensure we can easily track quantity and type of issues created so that we may report to the VA.gov community how VFS teams are doing.

Label structure:
![image.png](https://images.zenhubusercontent.com/5f62315e822e9b2aa88b1f1c/b6b01e47-d19c-434e-8891-a7c4267fc37b)

VFS Team labels:
![image.png](https://images.zenhubusercontent.com/5f62315e822e9b2aa88b1f1c/ee83d248-cf97-45bc-9347-11ffd4701e2b)
---
## Tasks
- [ ] _What work is necessary for this story to be completed?_

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by shiragoodman** [comment]
@JoeTice i added above our current labeling structure for all of the tickets we'll be creating from the new Staging Review process. This is what data we want to pull in for our future domo dashboards. Let me know when we can chat!

**Comment by shiragoodman** [comment]
- Who is the audience? OCTO-DE. As time goes on, we anticipate VFS PMs and VFS team members to gain interest.

   VFS PM view vs. OCTO-DE PO view - not really seeing much difference in the views, however want to note that this is for 2 different groups.

   
- Do you anticipate that there would be a benefit to showing OCTO-DE leadership or VFS team leadership (no good example offhand) some sort of visual representation of progress in the Collab Cycle? 

   Basically here is where the issue surfaced... at Staging Review the issue is open or closed.

**Comment by shiragoodman** [comment]
@peggygannon contact Analytics & Insights team to get Domo login access resolved.
@allison0034 open A[nalytics ticket ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=joanneesteban&labels=analytics-insights%2C+analytics-request%2C+access-request&template=analytics-request-google-analytics-domo-access.md&title=Request+access+to+Google+Analytics+and%2For+Domo)to request domo id

**Comment by shiragoodman** [comment]
per team meeting on 8/24: ![image.png](https://images.zenhubusercontent.com/5f62315e822e9b2aa88b1f1c/4b826ef9-aeae-4a86-8f22-ffe9956aa125)

**Comment by pjhill** [comment]
@shiragoodman Hey, I'm struggling a bit with the filter for these issues. Is this right?
We only care to collect issues that are `collab-cycle-feedback`, have a practice area of `QA`, `IA`, `Design`, `508/Accessibilty`, and be marked with a `CCIssueXX.YY` style label. Are those the only criteria? It looks like about 116 issues have been created that match those criteria. I have an example of the where clause in BigQuery below --
```
WHERE collab_cycle_feedback = 'collab-cycle-feedback'
    and practice_area in ('QA', 'IA', 'Content', 'Design', '508/Accessibility')
    and experience_standard like 'CCIssue%'
```

**Comment by shiragoodman** [comment]
thanks @pjhill  - yes, that's right... however the other label (that's not as necessary right now as it will be in the future) is `staging`. When we apply the new process to Midpoint review, the `staging` and `midpoint` labels will be very important.

**Comment by pjhill** [comment]
> thanks @pjhill - yes, that's right... however the other label (that's not as necessary right now as it will be in the future) is `staging`. When we apply the new process to Midpoint review, the `staging` and `midpoint` labels will be very important.

Cool, easy peasy -- for reference, I added another where clause to the query:
```
SELECT number, state, labels
FROM `vsp-analytics-and-insights.vsp_testing_tools.github_issues`,
    UNNEST(labels) as collab_cycle_feedback,
    UNNEST(labels) as practice_area,
    UNNEST(labels) as experience_standard,
    UNNEST(labels) as touchpoint
WHERE collab_cycle_feedback = 'collab-cycle-feedback'
    and practice_area in ('QA', 'IA', 'Content', 'Design', '508/Accessibility')
    and experience_standard like 'CCIssue%'
    and touchpoint in ('Staging','Midpoint')
```

**Comment by shiragoodman** [comment]
thanks! We may have to edit the query later on when we want to start tracking issues that are created at Midpoint and still are valid at Staging, or issues that are created at Midpoint then resolved by Staging, etc. This should be sufficient for now tho - thanks again!
